docs/src/logging.md

```diff
@@ -52,6 +52,53 @@ The log level for the operator can be configured by editing the `Deployment`
 definition of the operator and setting the `--log-level` command line argument
 to the desired value.
 
+!!! Important
+    Changing the log-level of the operator will not affect the log level of the
+    [instance manager](/instance_manager/) or other components.
+
+### Adding a custom log-level
```

Comment:
The first section, in my opinion, should handle modifying the existing value,
which is added by default in plain deployments.